app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -1,16 +1,26 @@
 .gem-c-intervention {
   @include govuk-text-colour;
-  @include govuk-responsive-padding(4);
-  @include govuk-responsive-margin(4, "bottom");
+  @include govuk-responsive-padding(6, "left");
+  @include govuk-responsive-padding(6, "right");
+  @include govuk-responsive-padding(4, "top");
+  @include govuk-responsive-padding(4, "bottom");
+  @include govuk-responsive-margin(6, "bottom");
   background-color: govuk-colour("light-grey", $legacy: "grey-4");
 }
 
 .gem-c-intervention__title {
-  @include govuk-font(24, $weight: bold);
+  @include govuk-font(19, $weight: bold);
```

Comment:
Not sure we need custom CSS for `.gem-c-intervention_title`. Could we not achieve this using [govuk-heading-s](https://design-system.service.gov.uk/styles/typography/headings/index.html)?

Reply:
Indeed. Done.